### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by josemigallas**
## Motivation

Maintain code base as tidy and consistent as possible.

## Description

Cleanup consist of: reformat code and remove unused imports.
Cleanup do not consist of: refactor code or change logic in any way.

### CODE REVIEW

**Comment by wtrocki** [comment]
> Maintain code base as tidy and consistent as possible.
-1 for doing that this way. Let's talk about standards on the list. Get required tooling into gradle and extend build to fail when changes do not conform with the standard. Making some manual changes are not improving overall state of the project because they are really presenting your point of view that most of the SDK developers will not be aware of and still push changes the old way.

**Comment by danielpassos** [comment]
@josemigallas Just wondering if you should not kill/merge too much cleanup commits in only one!

**Comment by wei-lee** [comment]
agree with @wtrocki. It's ok to merge this PR but I think it's more important to create a style/editor configuration to make sure we all use the same settings.

**Comment by danielpassos** [comment]
@wei-lee @wtrocki Why not use the default one provided by Android Studio/Google?

**Comment by josemigallas** [comment]
@wtrocki I see your point however this is not my point of view but the default Android Studio formatting rules, which I believe meets the most extended styling rules for Java.

@wei-lee That's true and also my intention. In the meanwhile I though why not updating the code for now.

**Comment by wei-lee** [comment]
@danielpassos I am using the default one in my Android Studio. So why do we still have the problem with formatting, are we missing a check/build step somewhere?

**Comment by danielpassos** [comment]
@wei-lee Because people forgot to apply it :bowtie:

My point is... we don't need to **create a style/editor configuration** we need to make sure it was applied when running ci

File: auth/src/main/java/org/aerogear/mobile/auth/AuthStateManager.java [hunk 1]
```diff
@@ -44,14 +46,15 @@ public synchronized void save(final OIDCCredentials authState) {
             clear();
         } else {
             SharedPreferences.Editor e = prefs.edit().putString(KEY_STATE, authState.serialize());
```
Comment (matzew) [inline auth/src/main/java/org/aerogear/mobile/auth/AuthStateManager.java]: make `e` final

File: auth/src/main/java/org/aerogear/mobile/auth/Callback.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package org.aerogear.mobile.auth;
 
 public interface Callback<T extends Object> {
```
Comment (matzew) [inline auth/src/main/java/org/aerogear/mobile/auth/Callback.java]: Let's add some javadoc

**Comment by josemigallas** [comment]
@wei-lee @wtrocki @danielpassos If everyone agrees, I propose closing this PR and opening a new one, with all the necessary changes to maintain code clean, that is: cleanup following just the default IDE formatting rules and necessary additions to CI tool to check lint and also some gradle task that formats code while building, if possible.

**Comment by matzew** [comment]
> also some gradle task that formats code while building

@josemigallas let's please not do that. This will cause large PRs, where lot's of distraction is on awfully formatted code.  The build should fail if things are not correctly formatted

**Comment by wei-lee** [comment]
@josemigallas +1. Let's run the checks during build time and fail it if there is code is not formatted/linted correctly.

**Comment by josemigallas** [comment]
OK then. Follow up task https://issues.jboss.org/browse/AGDROID-763

**Pull request closed without merging**